### PR TITLE
fix: change artifact id

### PR DESCRIPTION
**Pull request by solairerove**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,11 +4,11 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.github.nightcrawler</groupId>
```
Comment (karpukdm) [inline pom.xml]: maybe then <groupId>com.github.lostizalith</groupId>

Comment (solairerove) [reply pom.xml]: sure

Comment (solairerove) [reply pom.xml]: done

File: pom.xml [hunk 2]
```diff
@@ -3,12 +3,12 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.nightcrawler</groupId>
-    <artifactId>parent</artifactId>
+    <groupId>com.github.lostizalith</groupId>
```
Comment (karpukdm) [inline pom.xml:6]: did you change parent groupId and artifactId in pom.xml the ad-creator module?

**Review by karpukdm — APPROVED** [review]

**Pull request merged**